packages/richtext-lexical/src/exports/react/components/RichText/converter/converters/table.tsx

```diff
@@ -0,0 +1,55 @@
+import type {
+  SerializedTableCellNode,
+  SerializedTableNode,
+  SerializedTableRowNode,
+} from '../../../../../../nodeTypes.js'
+import type { JSXConverters } from '../types.js'
+
+export const TableJSXConverter: JSXConverters<
+  SerializedTableCellNode | SerializedTableNode | SerializedTableRowNode
+> = {
+  table: ({ node, nodesToJSX }) => {
+    const children = nodesToJSX({
+      nodes: node.children,
+    })
+    return (
+      <table className="lexical-table" style={{ borderCollapse: 'collapse' }}>
+        <tbody>{children}</tbody>
+      </table>
+    )
+  },
+  tablecell: ({ node, nodesToJSX }) => {
```

Comment:
is the `type` in Lexical called `tablecell`? it's not `td`?

Reply:
![CleanShot 2024-11-17 at 15 51 44@2x](https://github.com/user-attachments/assets/78140820-ffc4-4c84-a77c-8d84cc235c03)

correct, the node type here is tablecell